Remove SHOULD from 1 ACK/RTT (#211)
quicwg/ack-frequency: draft-ietf-quic-ack-frequency.md

the available bandwidth.

> To limit the consequences of reduced acknowledgement frequency, a sender

Reviewer: ```suggestion
To limit the consequences of reduced acknowledgement frequency, 
```

can cause a receiver to send an acknowledgement at least once per RTT when

> there are unacknowledged ack-eliciting packets in flight.

Author: ```suggestion
there are ack-eliciting packets in flight.
```

the available bandwidth.

To limit the consequences of reduced acknowledgement frequency, a sender

> can cause a receiver to send an acknowledgement at least once per RTT when

Author: ```suggestion
can cause a receiver to send an acknowledgement at least once per round trip when
```

> there are ack-eliciting packets in flight.

Author: ```suggestion
when there are ack-eliciting packets in flight.
```
